index.js

```diff
@@ -126,11 +152,23 @@ class HdKeyring extends EventEmitter {
 
   _initFromMnemonic (mnemonic) {
     this.mnemonic = mnemonic
-    const seed = bip39.mnemonicToSeed(mnemonic)
+    const seed = braveCrypto.passphrase.toBytes32(mnemonic)
     this.hdWallet = hdkey.fromMasterSeed(seed)
     this.root = this.hdWallet.derivePath(this.hdPath)
   }
 
+  /**
+   * Inits wallet from a given 32-byte random seed.
+   * @param {Uint8Array} seed
+   */
+  _initFromSeed (seed) {
```

Comment:
TODO: determine whether this needs a round of HKDF, per https://docs.google.com/document/d/1oMC2gWPTzybQLxgTC3JFSssfNxqo-OWII1m0QT2OUFw/edit?disco=AAAAC_BWLeQ